safe-global/safe-wallet-web
# fix: Disable network selector if social login wallet not supported

## What it solves

Part of #2452

## How this PR fixes it

- Enables the network selector but disables its items where social login is not supported instead
- Shows a tooltip in that case
- For non social login wallets everything stays the same

## How to test it

1. Open Safe
2. Open the network selector
3. Observe all items are enabled
4. Observe no Tooltip showing
5. Connect with MM
6. Step 2 and 3 are the same
7. Connect with Google
8. Observe the network selector is not disabled
9. Observe some items inside the network selector are disabled
10. Observe a tooltip

## Screenshots

![Screenshot 2023-10-23 at 18 01 19](https://github.com/safe-global/safe-wallet-web/assets/5880855/2647e049-6b3e-44f5-99e2-8ee07ff1b75b)


## Checklist
